src/sentry/static/sentry/app/components/activity/noteInput.jsx

```diff
@@ -206,6 +231,21 @@ const NoteInput = React.createClass({
     }
   },
 
+  getMemberData() {
+    let sessionUser = ConfigStore.get('user');
+    let results = [];
+    this.state.memberList.forEach(member => {
```

Comment:
It's also not ideal to have a low-level component like `NoteInput` communicating with so many global stores. Better to access these stores at a higher-level (e.g. the top-most view), and pass down the values as props. This makes testing more manageable.